Changes

```diff
@@ -119,6 +119,11 @@ Working version
 
 ### Compiler distribution build system:
 
+- GPR#1569: install bigarray,dynlink,graphics,raw_spacetime_lib,str,unix each
+  in a separate directory, and add these directories to the initial search path
+  unless -nostdincludes is given.
```

Comment:
I'm guessing you simply want the commit introducing the flag to be consistent about the plural vs singular, rather than necessarily use the singular version? Given that we have multiple include directories, it seems slightly better to keep the plural.